### DISCUSSION TITLE
Weekly Sync #20 - May 20, 2021

### DISCUSSION THREAD

**Discussion by turolopezsanabria** [SUI-Components/sui-components#1510] [Weekly Sync Streaming]
**Agenda**

- Weekly [Dashboard](https://pages.github.mpi-internal.com/scmspain/design-systems/sui/index.html)
- Colour and other tokens update
- Fields Group [discussion](https://github.com/SUI-Components/sui-components/discussions/1514)
- Drawer full-screen [discussion](https://github.com/SUI-Components/sui-components/discussions/1519)
- [Radio-button field](https://sui-components.vercel.app/workbench/molecule/radioButtonField/demo) adjustments in [this PR](https://github.mpi-internal.com/scmspain/frontend-ij--uilib-theme/pull/166/files)
- Modal by Bea
- Reset / Normalise: What's the best approach and location for it?
- Quick reminder deprecated [directory settings-compat-v7](https://github.com/SUI-Components/sui/tree/master/packages/sui-theme/src/settings-compat-v7)
- Component [upload](https://sui-components.vercel.app/workbench/atom/upload) iteration to allow the functionality **crop** and rotate. Updates on the [discussion](https://github.com/SUI-Components/sui-components/discussions/1501)
- We need a new Spacing variant between M & L

**Comment by ChrisDesignOps** [comment]
- We need a new [Spacing variant](https://github.com/SUI-Components/sui-theme/blob/02e8eaff4e701862d64331628f5956d2451a4ae3/src/settings/_spacing.scss) between M & L
